### PR TITLE
Add base components and styles

### PR DESCRIPTION
**Pull request by VikaScherban**
image

- add Counter component
- add SearchForm component
- add GenreSelect component

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div>
+          <Header></Header>
+          <Content></Content>
+          <Counter></Counter>
```
Comment (DmitriyDubravin) [inline src/App.tsx]: If a component has no children, you can close it like this: <Component />

File: src/components/header/search-form/SearchForm.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import './SearchForm.css';
+import {searchFunction} from "../../../models/search";
+import {useState} from "react";
+
+function SearchForm({ searchChange }: {searchChange: searchFunction}) {
+    const [formData, setFormData] = useState({
+        query: '',
+    });
```
Comment (DmitriyDubravin) [inline src/components/header/search-form/SearchForm.tsx:8]: Not only a callback but the Initial search query should be passed to the component as a property too.

File: src/components/header/search-form/SearchForm.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    return (
+      <form className="wrap-block" onSubmit={onSearch}>
+          <div className="label">FIND YOUR MOVIE</div>
+          <input type="text"
+                 name="query"
+                 value={formData.query}
+                 onChange={onQueryChange}
+                 onKeyDown={onKeyDown}
+                 onFocus={onQueryChange}
```
Comment (DmitriyDubravin) [inline src/components/header/search-form/SearchForm.tsx]: I don't need to change the state on input focus.

**Review by DmitriyDubravin — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (VikaScherban) [reply src/App.tsx]: Changed, thank you

File: src/components/header/search-form/SearchForm.tsx (hunk 2 above)
Comment (VikaScherban) [reply src/components/header/search-form/SearchForm.tsx:8]: Added, thank you

File: src/components/header/search-form/SearchForm.tsx (hunk 3 above)
Comment (VikaScherban) [reply src/components/header/search-form/SearchForm.tsx]: Removed, thank you

**Review by DmitriyDubravin — APPROVED** [review]

**Pull request merged**